File: Week_0.1/q5.c
#include <stdio.h>

int main() {
    float percent;

    printf("Enter Your Percentage : ");
    scanf("%f", &percent);

    if(percent >= 90) {
        printf("Your Grade : A\n");
    } else if(percent >= 80) {
        printf("Your Grade : B\n");
    } else if(percent >= 70) {
        printf("Your Grade : C\n");
    } else if( percent >= 60) {
        printf("Your Grade : D\n");
    } else if(percent >= 50) {
        printf("Your Grade : E\n");
    } else {
        printf("You are Failed\n");
    }
}

File: Week_3/q5.js
var readlineSync = require('readline-sync');

let mark = readlineSync.question("Enter your mark : ")
let grade = ""

if (mark >= 90) {
    grade = "A"
} else if (mark >= 80 && mark < 90) {
    grade = "B"
} else if (mark >= 70 && mark < 80) {
    grade = "C"
} else if (mark >= 60 && mark < 70) {
    grade = "D"
} else if (mark >= 50 && mark < 60) {
    grade = "E"
} else if (mark < 50) {
    console.log("Your are failed")
}

console.log("Your grade is : " +  grade);